src/App.tsx

```diff
@@ -12,18 +12,16 @@ function App() {
     <div className="App">
       <Header />
 
-      <div className="container">
-        <Routes>
-          <Route path="*" element={<PageNotFound />} />
-          <Route path="/" element={<HomePage />} />
-          <Route path="/home" element={<Navigate to="/" replace />} />
-          <Route path="/phones" element={<PhonesPage />}>
-            <Route index />
-            <Route path=":phoneId" element={<PhonesPage />} />
-          </Route>
-        </Routes>
-      </div>
-      
+      <Routes>
+        <Route path="*" element={<PageNotFound />} />
+        <Route path="/" element={<HomePage />} />
+        <Route path="/home" element={<Navigate to="/" replace />} />
+        <Route path="/phones" element={<PhonesPage />}>
+          <Route index />
+          <Route path=":phoneId" element={<PhonesPage />} />
+        </Route>
+      </Routes>
+
       <Footer />
```

Comment:
you could add use location and when location.pathname !== /burger-menu, set footer, same with header. cause they should not appear when burger menu open

Reply:
Will try to implement in next pull request